ClassNotFoundExceptions in jakarta branch

It is currently not possible (for me) to run the current _jakarta_ branch:

```
>mvn clean jetty:run -Pmyfaces40
[INFO] Scanning for projects...
[INFO]
[INFO] Using the MultiThreadedBuilder implementation with a thread count of 20
[INFO]
[INFO] -------------------< org.primefaces:primefaces-test >-------------------
[INFO] Building primefaces-test 1.0-SNAPSHOT
[INFO]   from pom.xml
[INFO] --------------------------------[ war ]---------------------------------
[INFO]
[INFO] --- clean:3.2.0:clean (default-clean) @ primefaces-test ---
[INFO]
[INFO] >>> jetty:12.0.3:run (default-cli) > test-compile @ primefaces-test >>>
[INFO]
[INFO] --- resources:3.3.1:resources (default-resources) @ primefaces-test ---
[INFO] skip non existing resourceDirectory C:\Users\username\Downloads\primefaces-test-jakarta\src\main\resources
[INFO] Copying 1 resource from src\main\webapp\WEB-INF to target
[INFO]
[INFO] --- compiler:3.11.0:compile (default-compile) @ primefaces-test ---
[INFO] Changes detected - recompiling the module! :source
[INFO] Compiling 3 source files with javac [debug release 11] to target\classes
[INFO]
[INFO] --- resources:3.3.1:testResources (default-testResources) @ primefaces-test ---
[INFO] skip non existing resourceDirectory C:\Users\username\Downloads\primefaces-test-jakarta\src\test\resources
[INFO]
[INFO] --- compiler:3.11.0:testCompile (default-testCompile) @ primefaces-test ---
[INFO] No sources to compile
[INFO]
[INFO] <<< jetty:12.0.3:run (default-cli) < test-compile @ primefaces-test <<<
[INFO]
[INFO]
[INFO] --- jetty:12.0.3:run (default-cli) @ primefaces-test ---
[INFO] Configuring Jetty for project: primefaces-test
[INFO] Classes = C:\Users\username\Downloads\primefaces-test-jakarta\target\classes
[INFO] Context path = /primefaces-test
[INFO] Tmp directory = C:\Users\username\Downloads\primefaces-test-jakarta\target\tmp
[INFO] web.xml file = file:///C:/Users/username/Downloads/primefaces-test-jakarta/target/web.xml
[INFO] Webapp directory = C:\Users\username\Downloads\primefaces-test-jakarta\src\main\webapp
[INFO] Web defaults = org/eclipse/jetty/ee10/webapp/webdefault-ee10.xml
[INFO] Web overrides =  none
[INFO] jetty-12.0.3; built: 2023-10-27T03:25:22.284Z; git: a873259fd5af141dedaa6c7fa8fe8fbdedce0d61; jvm 17.0.5+8-LTS
[WARNING] The XML schema [XMLSchema.dtd] could not be found. This is very likely to break XML validation if XML validation is enabled.
[WARNING] The XML schema [datatypes.dtd] could not be found. This is very likely to break XML validation if XML validation is enabled.
[WARNING] The XML schema [xml.xsd] could not be found. This is very likely to break XML validation if XML validation is enabled.
[INFO] CdiSpiDecorator enabled in ServletContext@oeje10mp.MavenWebAppContext@4c513c2c{/primefaces-test,/primefaces-test,b=[file:///C:/Users/username/Downloads/primefaces-test-jakarta/src/main/webapp/, jar:file:///C:/Users/username/.m2/repository/org/apache/myfaces/core/myfaces-impl/4.0.1/myfaces-impl-4.0.1.jar!/META-INF/resources/, jar:file:///C:/Users/username/.m2/repository/org/apache/myfaces/core/myfaces-api/4.0.1/myfaces-api-4.0.1.jar!/META-INF/resources/, jar:file:///C:/Users/username/.m2/repository/org/primefaces/primefaces/13.0.3/primefaces-13.0.3-jakarta.jar!/META-INF/resources/],a=STOPPED,h=oeje10s.SessionHandler@3068a1c9{STOPPED}}{file:///C:/Users/username/Downloads/primefaces-test-jakarta/src/main/webapp/}
Nov. 22, 2023 11:21:51 AM org.jboss.weld.environment.servlet.EnhancedListener onStartup
INFO: WELD-ENV-001008: Initialize Weld using ServletContainerInitializer
Nov. 22, 2023 11:21:51 AM org.jboss.weld.bootstrap.WeldStartup <clinit>
INFO: WELD-000900: 5.1.2 (Final)
Nov. 22, 2023 11:21:52 AM org.jboss.weld.bootstrap.WeldStartup startContainer
INFO: WELD-000101: Transactional services not available. Injection of @Inject UserTransaction not available. Transactional observers will be invoked synchronously.
Nov. 22, 2023 11:21:52 AM org.jboss.weld.event.ExtensionObserverMethodImpl checkRequiredTypeAnnotations
INFO: WELD-000411: Observer method [BackedAnnotatedMethod] org.apache.myfaces.config.annotation.CdiAnnotationProviderExtension.processAnnotatedType(@Observes ProcessAnnotatedType<T>) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Nov. 22, 2023 11:21:52 AM org.jboss.weld.environment.jetty.JettyContainer initialize
INFO: WELD-ENV-001213: Jetty CDI SPI support detected, CDI injection will be available in Listeners, Servlets and Filters.
Nov. 22, 2023 11:21:53 AM org.apache.myfaces.webapp.MyFacesContainerInitializer onStartup
INFORMATION: Using org.apache.myfaces.webapp.MyFacesContainerInitializer
[INFO] Session workerName=node0
[INFO] Started oeje10mp.MavenWebAppContext@4c513c2c{/primefaces-test,/primefaces-test,b=[file:///C:/Users/username/Downloads/primefaces-test-jakarta/src/main/webapp/, jar:file:///C:/Users/username/.m2/repository/org/apache/myfaces/core/myfaces-impl/4.0.1/myfaces-impl-4.0.1.jar!/META-INF/resources/, jar:file:///C:/Users/username/.m2/repository/org/apache/myfaces/core/myfaces-api/4.0.1/myfaces-api-4.0.1.jar!/META-INF/resources/, jar:file:///C:/Users/username/.m2/repository/org/primefaces/primefaces/13.0.3/primefaces-13.0.3-jakarta.jar!/META-INF/resources/],a=AVAILABLE,h=oeje10s.SessionHandler@3068a1c9{STARTED}}{file:///C:/Users/username/Downloads/primefaces-test-jakarta/src/main/webapp/}
Nov. 22, 2023 11:21:53 AM org.apache.myfaces.config.DefaultFacesConfigurationProvider getWebAppFacesConfig
INFORMATION: Reading config /WEB-INF/faces-config.xml
Nov. 22, 2023 11:21:53 AM org.apache.myfaces.util.ExternalSpecifications lambda$static$1
INFORMATION: MyFaces Core CDI support enabled
Nov. 22, 2023 11:21:53 AM org.apache.myfaces.util.ExternalSpecifications lambda$static$0
INFORMATION: MyFaces Core Bean Validation support disabled
Nov. 22, 2023 11:21:53 AM org.primefaces.webapp.PostConstructApplicationEventListener processEvent
INFORMATION: Running on PrimeFaces 13.0.3
Nov. 22, 2023 11:21:53 AM org.apache.myfaces.webapp.FacesInitializerImpl initFaces
INFORMATION: MyFaces Core has started, it took [817] ms.
[INFO] Started oeje10mp.MavenWebAppContext@4c513c2c{/primefaces-test,/primefaces-test,b=[file:///C:/Users/username/Downloads/primefaces-test-jakarta/src/main/webapp/, jar:file:///C:/Users/username/.m2/repository/org/apache/myfaces/core/myfaces-impl/4.0.1/myfaces-impl-4.0.1.jar!/META-INF/resources/, jar:file:///C:/Users/username/.m2/repository/org/apache/myfaces/core/myfaces-api/4.0.1/myfaces-api-4.0.1.jar!/META-INF/resources/, jar:file:///C:/Users/username/.m2/repository/org/primefaces/primefaces/13.0.3/primefaces-13.0.3-jakarta.jar!/META-INF/resources/],a=AVAILABLE,h=oeje10s.SessionHandler@3068a1c9{STARTED}}{file:///C:/Users/username/Downloads/primefaces-test-jakarta/src/main/webapp/}
[INFO] Started ServerConnector@4094adba{HTTP/1.1, (http/1.1)}{0.0.0.0:8080}
[INFO] Started oejs.Server@77a33246{STARTING}[12.0.3,sto=0] @7788ms
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  6.210 s (Wall Clock)
[INFO] Finished at: 2023-11-22T11:21:54+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.eclipse.jetty.ee10:jetty-ee10-maven-plugin:12.0.3:run (default-cli) on project primefaces-test: Error starting jetty: Error forming scan list: java.nio.file.NoSuchFileException: C:\Users\username\Downloads\primefaces-test-jakarta\src\main\conf\jetty-env.xml -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[←[1;34mINFO←[m] Stopped oejs.Server@77a33246{STOPPING}[12.0.3,sto=0]
[←[1;34mINFO←[m] Stopped oeje10mp.MavenWebAppContext@4c513c2c{/primefaces-test,/primefaces-test,b=[file:///C:/Users/username/Downloads/primefaces-test-jakarta/src/main/webapp/, jar:file:///C:/Users/username/.m2/repository/org/apache/myfaces/core/myfaces-impl/4.0.1/myfaces-impl-4.0.1.jar!/META-INF/resources/, jar:file:///C:/Users/username/.m2/repository/org/apache/myfaces/core/myfaces-api/4.0.1/myfaces-api-4.0.1.jar!/META-INF/resources/, jar:file:///C:/Users/username/.m2/repository/org/primefaces/primefaces/13.0.3/primefaces-13.0.3-jakarta.jar!/META-INF/resources/],a=AVAILABLE,h=oeje10s.SessionHandler@3068a1c9{STOPPED}}{file:///C:/Users/username/Downloads/primefaces-test-jakarta/src/main/webapp/}
[←[1;34mINFO←[m] Stopped ServerConnector@4094adba{HTTP/1.1, (http/1.1)}{0.0.0.0:8080}
Exception in thread "JettyShutdownThread" java.lang.NoClassDefFoundError: org/eclipse/jetty/io/ManagedSelector$CloseConnections
        at org.eclipse.jetty.io.ManagedSelector.doStop(ManagedSelector.java:135)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.stop(AbstractLifeCycle.java:132)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.stop(ContainerLifeCycle.java:182)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.doStop(ContainerLifeCycle.java:205)
        at org.eclipse.jetty.io.SelectorManager.doStop(SelectorManager.java:280)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.stop(AbstractLifeCycle.java:132)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.stop(ContainerLifeCycle.java:182)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.doStop(ContainerLifeCycle.java:205)
        at org.eclipse.jetty.server.AbstractConnector.doStop(AbstractConnector.java:370)
        at org.eclipse.jetty.server.AbstractNetworkConnector.doStop(AbstractNetworkConnector.java:82)
        at org.eclipse.jetty.server.ServerConnector.doStop(ServerConnector.java:241)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.stop(AbstractLifeCycle.java:132)
        at org.eclipse.jetty.ee10.maven.plugin.MavenServerConnector.doStop(MavenServerConnector.java:113)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.stop(AbstractLifeCycle.java:132)
        at org.eclipse.jetty.server.Server.doStop(Server.java:658)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.stop(AbstractLifeCycle.java:132)
        at org.eclipse.jetty.util.thread.ShutdownThread.run(ShutdownThread.java:136)
        Suppressed: java.lang.NoClassDefFoundError: org/eclipse/jetty/io/ManagedSelector$CloseConnections
                ... 17 more
        Caused by: java.lang.ClassNotFoundException: org.eclipse.jetty.io.ManagedSelector$CloseConnections
                ... 17 more
        Suppressed: java.lang.NoClassDefFoundError: org/eclipse/jetty/io/ManagedSelector$CloseConnections
                ... 17 more
        Caused by: java.lang.ClassNotFoundException: org.eclipse.jetty.io.ManagedSelector$CloseConnections
                ... 17 more
        Suppressed: java.lang.NoClassDefFoundError: org/eclipse/jetty/io/ManagedSelector$CloseConnections
                ... 17 more
        Caused by: java.lang.ClassNotFoundException: org.eclipse.jetty.io.ManagedSelector$CloseConnections
                ... 17 more
        Suppressed: java.lang.NoClassDefFoundError: org/eclipse/jetty/io/ManagedSelector$CloseConnections
                ... 17 more
        Caused by: java.lang.ClassNotFoundException: org.eclipse.jetty.io.ManagedSelector$CloseConnections
                ... 17 more
        Suppressed: java.lang.NoClassDefFoundError: org/eclipse/jetty/io/ManagedSelector$CloseConnections
                ... 17 more
        Caused by: java.lang.ClassNotFoundException: org.eclipse.jetty.io.ManagedSelector$CloseConnections
                ... 17 more
        Suppressed: java.lang.NoClassDefFoundError: org/eclipse/jetty/io/ManagedSelector$CloseConnections
                ... 17 more
        Caused by: java.lang.ClassNotFoundException: org.eclipse.jetty.io.ManagedSelector$CloseConnections
                ... 17 more
        Suppressed: java.lang.NoClassDefFoundError: org/eclipse/jetty/io/ManagedSelector$CloseConnections
                ... 17 more
        Caused by: java.lang.ClassNotFoundException: org.eclipse.jetty.io.ManagedSelector$CloseConnections
                ... 17 more
        Suppressed: java.lang.NoClassDefFoundError: org/eclipse/jetty/io/ManagedSelector$CloseConnections
                ... 17 more
        Caused by: java.lang.ClassNotFoundException: org.eclipse.jetty.io.ManagedSelector$CloseConnections
                ... 17 more
        Suppressed: java.lang.NoClassDefFoundError: org/eclipse/jetty/io/ManagedSelector$CloseConnections
                ... 17 more
        Caused by: java.lang.ClassNotFoundException: org.eclipse.jetty.io.ManagedSelector$CloseConnections
                ... 17 more
        Suppressed: java.lang.NoClassDefFoundError: org/eclipse/jetty/util/compression/CompressionPool$Entry
                at org.eclipse.jetty.util.compression.CompressionPool.doStop(CompressionPool.java:112)
                at org.eclipse.jetty.util.component.AbstractLifeCycle.stop(AbstractLifeCycle.java:132)
                at org.eclipse.jetty.util.component.ContainerLifeCycle.stop(ContainerLifeCycle.java:182)
                at org.eclipse.jetty.util.component.ContainerLifeCycle.doStop(ContainerLifeCycle.java:205)
                at org.eclipse.jetty.server.Handler$Abstract.doStop(Handler.java:480)
                at org.eclipse.jetty.server.Server.doStop(Server.java:669)
                ... 2 more
                Suppressed: java.lang.NoClassDefFoundError: org/eclipse/jetty/util/compression/CompressionPool$Entry
                        ... 8 more
                Suppressed: java.lang.NoClassDefFoundError: org/eclipse/jetty/ee10/plus/webapp/EnvConfiguration$Dumper
                        at org.eclipse.jetty.ee10.plus.webapp.EnvConfiguration.deconfigure(EnvConfiguration.java:153)
                        at org.eclipse.jetty.ee10.webapp.WebAppContext.stopContext(WebAppContext.java:1235)
                        at org.eclipse.jetty.ee10.servlet.ServletContextHandler.doStop(ServletContextHandler.java:1074)
                        at org.eclipse.jetty.ee10.maven.plugin.MavenWebAppContext.doStop(MavenWebAppContext.java:352)
                        at org.eclipse.jetty.util.component.AbstractLifeCycle.stop(AbstractLifeCycle.java:132)
                        at org.eclipse.jetty.util.component.ContainerLifeCycle.stop(ContainerLifeCycle.java:182)
                        at org.eclipse.jetty.util.component.ContainerLifeCycle.doStop(ContainerLifeCycle.java:205)
                        at org.eclipse.jetty.server.Handler$Abstract.doStop(Handler.java:480)
                        ... 7 more
                Caused by: java.lang.ClassNotFoundException: org.eclipse.jetty.ee10.plus.webapp.EnvConfiguration$Dumper
                        at org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy.loadClass(SelfFirstStrategy.java:50)
                        at org.codehaus.plexus.classworlds.realm.ClassRealm.unsynchronizedLoadClass(ClassRealm.java:271)
                        at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:247)
                        at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:239)
                        ... 15 more
                Suppressed: java.lang.NoClassDefFoundError: org/eclipse/jetty/util/thread/ReservedThreadExecutor$ReservedThread
                        at org.eclipse.jetty.util.thread.ReservedThreadExecutor.doStop(ReservedThreadExecutor.java:196)
                        at org.eclipse.jetty.util.component.AbstractLifeCycle.stop(AbstractLifeCycle.java:132)
                        at org.eclipse.jetty.util.component.ContainerLifeCycle.stop(ContainerLifeCycle.java:182)
                        at org.eclipse.jetty.util.component.ContainerLifeCycle.doStop(ContainerLifeCycle.java:205)
                        at org.eclipse.jetty.util.thread.QueuedThreadPool.doStop(QueuedThreadPool.java:236)
                        ... 7 more
                Caused by: java.lang.ClassNotFoundException: org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread
                        at org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy.loadClass(SelfFirstStrategy.java:50)
                        at org.codehaus.plexus.classworlds.realm.ClassRealm.unsynchronizedLoadClass(ClassRealm.java:271)
                        at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:247)
                        at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:239)
                        ... 12 more
        Caused by: java.lang.ClassNotFoundException: org.eclipse.jetty.util.compression.CompressionPool$Entry
                at org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy.loadClass(SelfFirstStrategy.java:50)
                at org.codehaus.plexus.classworlds.realm.ClassRealm.unsynchronizedLoadClass(ClassRealm.java:271)
                at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:247)
                at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:239)
                ... 8 more
Caused by: java.lang.ClassNotFoundException: org.eclipse.jetty.io.ManagedSelector$CloseConnections
        at org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy.loadClass(SelfFirstStrategy.java:50)
        at org.codehaus.plexus.classworlds.realm.ClassRealm.unsynchronizedLoadClass(ClassRealm.java:271)
        at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:247)
        at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:239)
        ... 17 more
```

--- a/pom.xml
+++ b/pom.xml
@@ -100,7 +100,6 @@
                     <!-- see https://eclipse.dev/jetty/documentation/jetty-12/programming-guide/index.html#jetty-run-goal -->
                     <webApp>
                         <contextPath>/primefaces-test</contextPath>
-                        <jettyEnvXml>${project.basedir}/src/main/conf/jetty-env.xml</jettyEnvXml>
                     </webApp>
                     <scan>5</scan>
                     <webXml>${project.build.directory}/web.xml</webXml>

